src/compute/src/server.rs

```diff
@@ -450,54 +434,40 @@ impl<'w, A: Allocate> Worker<'w, A> {
             let cmd_queue = Rc::clone(&cmd_queue);
 
             self.timely_worker.dataflow::<u64, _, _>(move |scope| {
-                source(scope, "CmdSource", |capability, info| {
-                    // Send activator for this operator back
-                    let activator = scope.sync_activator_for(&info.address[..]);
-                    activator_tx.send(activator).expect("activator_tx working");
-
-                    //Hold onto capbility until we receive a disconnected error
-                    let mut cap_opt = Some(capability);
-                    // Drop capability if we are not the leader, as our queue will
-                    // be empty and we will never use nor importantly downgrade it.
+                let mut pump_op = AsyncOperatorBuilder::new("CmdSource".to_string(), scope.clone());
+
+                let (mut cmd_output, cmd_stream) = pump_op.new_output();
+
+                let _shutdown_button = pump_op.build(move |mut capabilities| async move {
```

Comment:
if you don't use the button you can simply ignore it and not bind it to a variable, it's not marked as a `#[must_use]` I think